src/test/librgw_file.cc

```diff
@@ -106,14 +106,11 @@ TEST(LibRGW, LOOKUP_BUCKETS) {
   if (! fs)
     return;
 
-  int ret = 0;
   for (auto& fid_row : bucket_matrix) {
     auto& fid = get<0>(fid_row);
     // auto& obj_vector = get<1>(fid_row);
     struct rgw_file_handle *rgw_fh = nullptr;
-    ret = rgw_lookup(fs, fs->root_fh, get<0>(fid).c_str(), &rgw_fh,
-		    0 /* flags */);
-    ASSERT_EQ(ret, 0);
+    ASSERT_EQ(0, rgw_lookup(fs, fs->root_fh, get<0>(fid).c_str(), &rgw_fh, 0 /* flags */));
```

Comment:
Not that I want to be nit-picking, but the source has at least ~17000 lines of code that exceed the 80 char limit. Is this something we start to enforce now? But I will change it anyway if this is what you want.